packages/jest-expect/src/index.ts

```diff
@@ -5,20 +5,19 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import type {Config} from '@jest/types';
-import {Expect, expect} from 'expect';
+import {expect} from 'expect';
 import {
   addSerializer,
   toMatchInlineSnapshot,
   toMatchSnapshot,
   toThrowErrorMatchingInlineSnapshot,
   toThrowErrorMatchingSnapshot,
 } from 'jest-snapshot';
+import type {JestExpect} from './types';
 
-export default function jestExpect(
-  config: Pick<Config.GlobalConfig, 'expand'>,
-): Expect {
-  expect.setState({expand: config.expand});
+export type {JestExpect} from './types';
+
+export function createJestExpect(): JestExpect {
```

Comment:
I meant `expect` can export a `createJestExpect` that we use here so we don't mutate the shared `expect` with our `extend`. But since `extend` work by using module state instead of internal to some closure that doesn't make sense.

But I don't think we need to export this one (I like keeping it, tho, for internal use)